deprecations/README.md

```diff
@@ -35,6 +35,8 @@ The table columns have the following meanings:
 
 | Feature | Comments | Deprecated | Discontinued |
 | :-- | :-- | :-- | :-- |
+| `BasePrices`<br>in [Get rate pricing](../operations/rates.md#get-rate-pricing) | Replaced by `BaseAmountPrices` | 26 Oct 2022 | 26 Apr 2022 |
+| `Prices`<br>in [Resource category pricing](../operations/rates.md#resource-category-pricing) | Replaced by `AmountPrices` | 26 Oct 2022 | 26 Apr 2022 |
```

Comment:
@MikeAdamsMews Again here `Prices` are nested inside an object that is in the response. Is it ok like this?

Reply:
Thanks for updating the Deprecations page. Yes, I think that's clear, the user can click on the link and find the reference ok, so I don't see any problem. Nice to see the 'discontinued' column being used! 😄